Tests/BackendIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -466,6 +488,17 @@ private extension StoreKit1IntegrationTests {
         return data
     }
 
+    @discardableResult
+    func purchaseConsumablePackage(
+        file: FileString = #file,
+        line: UInt = #line
+    ) async throws -> PurchaseResultData {
+        let offering = try await XCTAsyncUnwrap(try await Purchases.shared.offerings().offering(identifier: "coins"))
+        let package = try XCTUnwrap(offering.package(identifier: "10.coins"))
```

Comment:
I wonder if the offering id and package id should be parameters of the method... Otherwise, maybe we can  extract them to constants.

Reply:
We only have 1 consumable for now in the config file, so for now I'd say YAGNI.
But I'll extract to constants.